interface/src/api.ts

```diff
@@ -0,0 +1,216 @@
+const CURRENT_API_VERSION = "0.1.0";
+
+/**
+ * This class represents an extension that can be installed with the extension-manager.
+ *
+ * Wondering why we picked TypeScript as an interface? Check the design.md / Extension API
+ */
+export interface ExasolExtension {
+    /** Name of the extension */
+    name: string;
+    /** Description of the extension */
+    description: string;
+    /** Files that this extension requires in BucketFS. */
+    bucketFsUploads?: BucketFSUploads;
```

Comment:
Is there a reason why you use a map instead of an array? I think an array would be easier to understand.
If necessary you can put the key as an ID into the `BucketFSUpload` interface.

Reply:
I copied that from your design ;-)

Reply:
I can change it.